extensions/amp-pan-zoom/1.0/amp-pan-zoom.js

```diff
@@ -0,0 +1,35 @@
+import {isExperimentOn} from '#experiments';
+
+import {AmpPreactBaseElement, setSuperClass} from '#preact/amp-base-element';
+
+import {userAssert} from '#utils/log';
+
+import {BaseElement} from './base-element';
+
+import {CSS} from '../../../build/amp-pan-zoom-1.0.css';
+
+/** @const {string} */
+const TAG = 'amp-pan-zoom';
+
+class AmpPanZoom extends setSuperClass(BaseElement, AmpPreactBaseElement) {
```

Comment:
Is this unneeded now?

Reply:
I don't think this is needed anymore, but this PR was started before that decision, so it's easier to just leave this here along with the tests and storybook.  There are several Bento components that still have these amp-* files, so we should probably clean them all up in one fell swoop.